package.json

```diff
@@ -29,12 +29,12 @@
     "@types/react-dom": "^16.9.0",
     "@types/react-virtualized": "^9.21.2",
     "@types/seedrandom": "^2.4.28",
-    "@types/storybook__addon-knobs": "^5.0.3",
```

Comment:
note that the latest version of storybook provides it's own types https://www.npmjs.com/package/@types/storybook__addon-knobs